app/Models/Foundation/Main/IGroup.php

```diff
@@ -31,4 +31,5 @@ interface IGroup
     const SummitRoomAdministrators = 'summit-room-administrators';
     const SummitRegistrationAdmins = 'summit-registration-administrators';
     const Testers                  = 'tester-groups';
+    const Sponsors                 = 'sponsors';
```

Comment:
you created the group
but u need to add the group to the auth group collection on the endpoint seeding too
like here
https://github.com/OpenStackweb/summit-api/blob/a02088453cff6fb953998a34f15f66a9d107ce17/database/seeders/ApiEndpointsSeeder.php#L2011
please review all endpoints authz group updates and depicted on the ticket, if they have authz group
this new group should be added